packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx

```diff
@@ -77,6 +79,18 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
                         >
                             <Selfie selfie={steps_state.selfie} onSelect={setSelfie} />
                         </Wizard.Step>
+                        <Wizard.Step
+                            title='Address verification'
+                            is_submit_disabled={steps_state.address_verification_disabled}
```

Comment:
@hamid-deriv could we add is_ to address_verification_disabled property

Reply:
Sorted.